vms/avm/vm_test.go

```diff
@@ -629,15 +629,26 @@ func TestGenesisGetUTXOs(t *testing.T) {
 // - Fetching all UTXOs when they exceed maxUTXOsToFetch (1024)
 func TestGenesisGetPaginatedUTXOs(t *testing.T) {
 	addr0Str, _ := formatting.FormatBech32(testHRP, addrs[0].Bytes())
+	addr1Str, _ := formatting.FormatBech32(testHRP, addrs[1].Bytes())
+	addr2Str, _ := formatting.FormatBech32(testHRP, addrs[2].Bytes())
 
-	// Create a starting point of 2000 UTXOs
+	// Create a starting point of 2000 UTXOs on different addresses
 	utxoCount := 2000
```

Comment:
Can we replace `2000` with `maxUTXOsToFetch + [some amount]` so that this test remains valid if `maxUTXOsToFetch` changes?